docs/en/edge/guide/platforms/android/index.md

```diff
@@ -29,11 +29,12 @@ Overview.  For details on the CLI, see The Command-Line Interface.
 
 ## Requirements and Support
 
-Cordova for Android requires the Android SDK. See the Android SDK's
-[System Requirements](http://developer.android.com/sdk/index.html).
+Cordova for Android requires the Android SDK which could be installed
+on OS X, Linux or Windows operation system. See the Android SDK's
+[System Requirements](http://developer.android.com/sdk/index.html#Requirements).
 
 Cordova supports Android 2.3.x (Gingerbread, starting with Android API level 10)
```

Comment:
I decided to just merge it in. I can fix it on master myself. Can you
comment on the mailing list thread "Deprecation Wars. ICS vs Gingerbread"

On Wed, Apr 22, 2015 at 12:07 PM, Sergey Grebnov <notifications@github.com>
wrote:

> In docs/en/edge/guide/platforms/android/index.md
> <https://github.com/apache/cordova-docs/pull/279#discussion_r28904464>:
>
> > @@ -29,11 +29,12 @@ Overview.  For details on the CLI, see The Command-Line Interface.
> >
> >  ## Requirements and Support
> >
> > -Cordova for Android requires the Android SDK. See the Android SDK's
> > -[System Requirements](http://developer.android.com/sdk/index.html).
> > +Cordova for Android requires the Android SDK which could be installed
> > +on OS X, Linux or Windows operation system. See the Android SDK's
> > +[System Requirements](http://developer.android.com/sdk/index.html#Requirements).
> >
> >  Cordova supports Android 2.3.x (Gingerbread, starting with Android API level 10)
>
> Thx Steve, will double check, but I still see minSdkVersion is 10 in
> platform template
> < uses-sdk android:minSdkVersion="10" />
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/cordova-docs/pull/279/files#r28904464>.
>